packages/playground/package.json

```diff
@@ -63,30 +59,36 @@
     "@uiw/react-markdown-preview": "^5.1.1",
     "caret-xy": "^2.0.3",
     "clipboard-copy": "^3.1.0",
+    "diff2html": "3.4.22",
+    "fast-diff": "^1.3.0",
+    "highlight.js": "^11.9.0",
     "lodash-es": "^4.17.21",
     "lucide-react": "^0.407.0",
     "match-sorter": "^6.3.1",
     "moment": "^2.29.4",
     "monaco-editor": "0.50.0",
     "monaco-yaml": "5.2.1",
+    "overlayscrollbars": "^2.10.0",
+    "pluralize": "^8.0.0",
     "prettier": "^3.0.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-hook-form": "7.53.0",
     "react-keywords": "^1.0.0",
-    "yaml": "^2.5.0",
-    "yaml-loader": "^0.8.1",
-    "zod": "^3.23.8",
+    "react-timeago": "^7.2.0",
     "rehype-external-links": "^3.0.0",
     "rehype-sanitize": "^6.0.0",
     "rehype-video": "^2.0.2",
-    "pluralize": "^8.0.0"
+    "yaml": "^2.5.0",
+    "yaml-loader": "^0.8.1",
+    "zod": "^3.23.8"
   },
   "devDependencies": {
-    "@types/pluralize": "^0.0.33",
     "@types/lodash-es": "^4.17.9",
+    "@types/pluralize": "^0.0.33",
     "@types/react": "^18.2.21",
     "@types/react-dom": "^18.2.7",
+    "@types/react-timeago": "^4.1.7",
```

Comment:
https://harness.atlassian.net/browse/SECAPP-1425

Reply:
@sans-harness where are we still using `react-timeago`? We should try to remove all it's references. 

We do have some utils already added here https://github.com/harness/canary/blob/pr-tab-as-hook/packages/playground/src/utils/utils.ts#L79.

Reply:
removed `react-timeago`